Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch-slim
+FROM debian:bullseye-slim
```

Comment:
Should we align it with the base OS version of GHES and use `buster` instead?

Reply:
We certainly can do that. I sort of went back and forth on this, and went with the newest. 

On the other hand, we're running tests here in `ubuntu` vs Debian so we are not super tightly coupled to a particular distro/release.